tdrs-backend/tdpservice/search_indexes/parsers/validators/validator.py

```diff
@@ -0,0 +1,42 @@
+from cerberus import Validator, errors
+
+class FatalEditWarningsValidator(Validator):
+    """Validator for TANF and SSP fatal edit warnings."""
+
+    definitions = {
+        'gt': 'greater than',
+        'lt': 'less than',
+        'gte': 'greater than or equal to',
+        'lte': 'less than or equal to',
+        'in': 'in',
+    }
+
```

Comment:
I'm not entirely clear on the use cases that drove this over using the [Cerberus built-in min,max checks](https://docs.python-cerberus.org/en/stable/validation-rules.html#min-max). What was the value-add here? Open to hearing your viewpoint

Reply:
I did this so that we could edit the error objects, but in the end I had to construct those manually anyways so we are actually getting anything from this currently. 

However, I would consider leaving this for now just in case we explore doing the error handling in Cerberus. Perhaps we could make a follow up ticket to evaluate this work and determine if we should go deeper into Cerberus, or remove the aspects that we are not utilizing, like the `_validate_gt` vs 'max'. Every function used _construct_error_obj so there is potential to modify that later.

Also it was easier to visualize having a `_validate_gt` and `_validate_gte` function in the same location rather than using some custom validators and some default ones.

Reply:
I also thing `gt`and `lt` are clearer then `min` and `max` when reading the schema.